gradle.properties

```diff
@@ -1,11 +1,11 @@
-profileServiceVersion=5.2-SNAPSHOT
-grailsVersion=5.2.4
-gorm.version=7.3.2
-groovyVersion=3.0.11
-mongodbDriverVersion=4.5.1
-grailsGradlePluginVersion=5.2.3
-gormMongoVersion=7.3.0
-alaSecurityLibsVersion=6.2.0
+profileServiceVersion=6.0-SNAPSHOT
+grailsVersion=6.2.3
+#gorm.version=7.3.2
+#groovyVersion=3.0.11
+mongodbDriverVersion=4.11.1
```

Comment:
Just checking, i think grails-gorm-mongodb will pull in 4.11.5 - is there a reason you've specified 4.11.1?

Reply:
updated

Reply:
ta